### PR TITLE
header nav bar

### PR DESCRIPTION
**Pull request by sandikbarr**
Moves the header nav bar to the style guide.

### CODE REVIEW

File: src/demo/styles.css [hunk 1]
```diff
@@ -2,6 +2,38 @@
 @import 'node_modules/pikaday/css/pikaday.css';
 @import 'node_modules/pikaday/css/triangle.css';
 
+html {
+  line-height: 1.5em;
+  height: 100%;
+  background-color: #252525;
+  background-image: url("data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%229%22%20height%3D%2218%22%20viewBox%3D%220%200%205%2010%22%3E%3Crect%20width%3D%22110%25%22%20x%3D%22-5%25%22%20y%3D%22-5%25%22%20height%3D%22110%25%22%20fill%3D%22none%22%2F%3E%3Cpath%20stroke%3D%22%23000%22%20stroke-width%3D%22.62%22%20d%3D%22M-2%201l9%209M-2%206l9%209M-2-4l9%209%22%2F%3E%3C%2Fsvg%3E");
```
Comment (sandikbarr) [inline src/demo/styles.css]: This is the only way we have to deal with images right now

File: src/demo/styles.css [hunk 2]
```diff
@@ -2,6 +2,38 @@
 @import 'node_modules/pikaday/css/pikaday.css';
 @import 'node_modules/pikaday/css/triangle.css';
```
Comment (sandikbarr) [inline src/demo/styles.css]: These styles are pulled from publish. However, they aren't able to be used in publish from the styleguide at this time. We don't have any plans currently for exporting global styles from the styleguide, which is mostly what the original webpack build was supposed to be doing for us.

**Comment by sandikbarr** [comment]
I'll add something for smaller screen sizes, haven't ported over the image loader yet though.

File: src/demo/app/toc.component.ts [hunk 3]
```diff
@@ -4,8 +4,29 @@ import { Component } from '@angular/core';
...
   template: `
     <section>
+      <h2>Navigation</h2>
+      <p>In the header above you see the following components in use</p>
+      <ul>
+        <li>HeaderComponent 
+          - a fixed position navigation bar that shows a home logo link and supports other navigation items with projected content</li>
+        <li>NavItemComponent
+          - a component that provides a router link within the application or an href link outside the application dependent 
+          on it's Inputs <code>route</code>, <code>href</code>, and <code>text</code></li>
```
Comment (gcampo88) [inline src/demo/app/toc.component.ts]: it's Inputs should be its Inputs (or maybe its `Input`s ?)

**Pull request merged**